### PR TITLE
signin, signup, signout and logo appear in Header with functionality

### PR DESCRIPTION
**Pull request by kmsmith5**

### CODE REVIEW

File: app/javascript/components/components/Footer.js
Comment (halebeau) [inline app/javascript/components/components/Footer.js]: Main branch should not have commented code. Make sure that it's cleaned up before you PR.

File: app/javascript/components/App.js
Comment (elysemontano) [inline app/javascript/components/App.js]: Please remove console.logs when submitting pr's

**Review by elysemontano — APPROVED** [review]
A few small things, please make sure to remove console logs before merging to main and remove unnecessary commented out code.  Please do this before merging, but approving so this is not a blocker

**Review by elysemontano — APPROVED** [review]
Changes look great!

**Pull request merged**
